src/alvarium/annotators/handler/factories.py

```diff
@@ -0,0 +1,13 @@
+from requests import Request
+from alvarium.annotators.handler.ed25519 import Ed25519RequestHandler
+from alvarium.sign.contracts import SignInfo, SignType
+from .interfaces import RequestHandler
+from .exceptions import RequestHandlerException
+
+class RequestHandlerFactory():
+
+    def getRequestHandler(self, request: Request, keys: SignInfo) -> RequestHandler:
```

Comment:
Shouldn't we implement a `MOCK` type request handler for the sake of unit testing?